### DISCUSSION TITLE
Stable releases

### DISCUSSION THREAD

**Discussion by somebody1234** [icsharpcode/AvaloniaILSpy#75] [Ideas]
any chance we could get stable releases? specifically, artifacts can't be downloaded via curl, and arch .net core is on 3.1.15, which is newer than the latest build on master

**Comment by christophwille** [comment]
There is no "stable" per se, but there is a release https://github.com/icsharpcode/AvaloniaILSpy/releases/tag/v7.0-rc1

**Reply by somebody1234** [reply]
yea, small problem is that that's on 3.1.14; 3.1.15 was released on 11 may. no big deal tho since arch can build from source fine
